script/init.js

```diff
@@ -95,3 +95,16 @@ Hooks.once("ready", () => {
 Hooks.once("diceSoNiceReady", (dice3d) => {
   registerDiceSoNice(dice3d);
 });
+
+Hooks.on("renderChatMessage", async (app, html, msg) => {
+
+  // Add drag and drop functonality to posted items
+  let postedItem = html.find(".chat-item")[0];
+  if (postedItem)
+  {
+    postedItem.setAttribute("draggable", true);
```

Comment:
I would suggest also adding the `draggable` class to `postedItem` here. The `draggable` class is defined by the Foundry stylesheets and gives us a `move` cursor, to better communicate to the user that it is draggable.

Reply:
Good idea! I should definitely apply that to my other systems too.